tests/src/test/scala/com/nvidia/spark/rapids/unit/StringUnitTest.scala

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.unit
+
+import ai.rapids.cudf.{ColumnVector, HostColumnVector}
+import com.nvidia.spark.rapids.{GpuUnitTests, GpuColumnVector, GpuScalar, CudfTestHelper}
+import org.apache.spark.sql.rapids.GpuStringRepeat
+import org.apache.spark.sql.functions.{col, lit}
+import org.apache.spark.sql.types.DataTypes
+
+class StringUnitTest extends GpuUnitTests {
```

Comment:
IMO this should be renamed to `StringRepeatSuite` (and the file renamed accordingly).  We don't want all string tests to be collected in this file.

Reply:
Should be `StringRepeatUnitTest`, right? I observed that `*Suite.scala` are all for expr testing. The unit tests are in `*UnitTest.scala` files under the `unit` folder.

Reply:
Sure, that works too.  I'm used to scala tests being called a `Suite` because that's how scalatest refers to them.  "Suite" does not preclude being a unit test, but if that's how other things in this package want to be called, so be it.